internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer.go

```diff
@@ -201,8 +201,8 @@ func (*Sequencer) Boot(r runtime.Runtime) []runtime.Phase {
 	phases := PhaseList{}
 
 	wipe := procfs.ProcCmdline().Get(constants.KernelParamWipe).First()
-	if wipe != nil && *wipe == "system" {
-		return phases.Append("wipeSystemDisk", ResetSystemDisk).Append("reboot", Reboot)
+	if wipePhases := parseWipeParamString(wipe); wipePhases != nil {
```

Comment:
I left a comment here in the previous version that we need to consider moving this earlier to the `Initialize` sequence (e.g. before `mountStatePartition`),  so that wipe can work always, and it should be considerably easier.

Reply:
Moved just above `MountStatePartition`.